### PR TITLE
Fix mini typos in docs

### PR DESCRIPTION
**Pull request by svilupp**
As part of the JuliaCon hackathon, I have run all the docs through Grammarly. In case of ambiguity (eg, inplace vs in-place), I have googled for the more popular spelling (often picked based on Wikipedia).

There were no changes to active code, ie, only comments or docs were changed.

Examples:
- log-scale --> on a log scale
- inplace --> in-place
- sub type (was inconsistently used) --> subtype

I'm happy to withdraw this PR if it's not helpful.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ A Julia package for probability distributions and associated functions. Particul
 * Moments (e.g mean, variance, skewness, and kurtosis), entropy, and other properties
 * Probability density/mass functions (pdf) and their logarithm (logpdf)
 * Moment generating functions and characteristic functions
-* Sampling from population or from a distribution
+* Sampling a population or a distribution
```
Comment (devmotion) [inline README.md:16]: I guess this change should be reverted as we're not sampling populations or distributions but drawing samples from a population or distribution:
```suggestion
* Sampling from a population or from a distribution
```

File: docs/src/multivariate.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ pdf(::MultivariateDistribution, ::AbstractArray)
 logpdf(::MultivariateDistribution, ::AbstractArray)
 loglikelihood(::MultivariateDistribution, ::AbstractVector{<:Real})
 ```
-**Note:** For multivariate distributions, the pdf value is usually very small or large, and therefore direct evaluating the pdf may cause numerical problems. It is generally advisable to perform probability computation in log-scale.
+**Note:** For multivariate distributions, the pdf value is usually very small or large, and therefore direct evaluation of the pdf may cause numerical problems. It is generally advisable to perform probability computation on a log scale.
```
Comment (devmotion) [inline docs/src/multivariate.md:38]: Hmm, not sure about "on a log scale". I guess I would prefer the original version:
```suggestion
**Note:** For multivariate distributions, the pdf value is usually very small or large, and therefore direct evaluation of the pdf may cause numerical problems. It is generally advisable to perform probability computation in log scale.
```

File: src/univariate/continuous/skewedexponentialpower.jl [hunk 3]
```diff
@@ -74,7 +74,7 @@ function m_k(d::SkewedExponentialPower, k::Integer)
         loggamma(inv_p) + log(abs((-1)^k * α^(1 + k) + (1 - α)^(1 + k)))
 end
 
-# needed for odd moments on log-scale
+# needed for odd moments on a log scale
```
Comment (devmotion) [inline src/univariate/continuous/skewedexponentialpower.jl:77]: Same here:
```suggestion
# needed for odd moments in log scale
```

**Review by devmotion — COMMENTED** [review]
Thanks for the PR! Seems to be an improvement, I only had some minor comments :slightly_smiling_face:

File: docs/src/multivariate.md (hunk 2 above)
Comment (svilupp) [reply docs/src/multivariate.md]: "on" is a much more popular preposition in this case, but happy to take your suggestion

**Review by devmotion — APPROVED** [review]
Thanks for the PR!

**Pull request merged**